src/lib/cli_lib/commands.ml

```diff
@@ -137,9 +137,15 @@ let validate_transaction =
     if !num_fails > 0 then (
       Format.printf "Some transactions failed to verify@." ;
       exit 1 )
-    else (
-      Format.printf "All transactions were valid@." ;
-      exit 0 ) )
+    else
+      let[@alert "--deprecated"] first = Caml.Stream.peek jsons in
+      match first with
+      | None ->
+          Format.printf "Empty transactions list is not supported@." ;
```

Comment:
Nit: `could not parse any transactions`?

Reply:
Makes more sense, thank you.